packages/notifications-controller/src/types/on-chain-notification/schema.d.ts

```diff
@@ -0,0 +1,300 @@
+/**
+ * This file was auto-generated by openapi-typescript.
```

Comment:
How does this file get auto-generated? And how does it get used?

Reply:
Yep, this was generated from our OpenAPI spec.

Noted & Future Comment/addition - add in the [Open API Types generator script](https://www.npmjs.com/package/openapi-typescript).
- This does not need to be run on every build for CI. It can be co-ordinated with our team.